renderdoc/driver/d3d12/d3d12_device_wrap.cpp

```diff
@@ -747,7 +755,7 @@ bool WrappedID3D12Device::Serialise_CreateReservedResource(
 bool WrappedID3D12Device::Serialise_CreateHeap(const D3D12_HEAP_DESC *pHeapDesc,
                                                REFIID riidResource, void **ppvResource)
 {
-  SERIALISE_ELEMENT(D3D12_HEAP_DESC, desc, *pHeapDesc);
+  SERIALISE_ELEMENT_PTR(D3D12_HEAP_DESC, desc, pHeapDesc);
```

Comment:
FWIW I think this change is redundant but fine to make. The `SERIALISE_ELEMENT_PTR` is kind of leftovers, and would only really be useful if you had some pointer that could be NULL and you wanted a default-initialised value instead. Here, `pHeapDesc` is required.